using-mobile-extensions/adobe-campaign-standard-beta.md

```diff
@@ -85,23 +85,23 @@ In Xcode, import the Mobile Core beta, Campaign Standard beta, and Profile exten
 #### Objective-C
 
 ```objectivec
-#import <ACPCore_iOS/ACPCore_iOS.h>
-#import <ACPCampaign_iOS/ACPCampaign_iOS.h>
-#import <ACPIdentity_iOS/ACPIdentity_iOS.h>
-#import <ACPSignal_iOS/ACPSignal_iOS.h>
-#import <ACPLifecycle_iOS/ACPLifecycle_iOS.h>
-#import <ACPUserProfile_iOS/ACPUserProfile.h>
+#import "ACPCore.h"
```

Comment:
Since we are not releasing Campaign Standard Beta with static lib changes, I think we need not update this file.